### PR TITLE
[wasm] fix perf after test renames

### PR DESCRIPTION
**Pull request by pavelsavara**
fix perf after renames in https://github.com/dotnet/runtime/pull/61596

### CODE REVIEW

File: docs/articles/configs/toolchains.md [hunk 1]
```diff
@@ -339,7 +339,7 @@ And FINALLY build Mono Runtime with Web Assembly support:
 Before you run the benchmarks, you need to make sure that following two file exists:
 
 ```cmd
-runtime/src/mono/wasm/runtime-test.js
```
Comment (naricc) [inline docs/articles/configs/toolchains.md:342]: I think we actually need to delete this; the documentation is old. We no longer directly specify the path to this file.

**Comment by naricc** [comment]
@adamsitnik

**Review by naricc — APPROVED** [review]

**Comment by pavelsavara** [comment]
I don't have permissions to merge this @naricc , could you please do it my behalf ?

**Review by adamsitnik — APPROVED** [review]
LGTM, thank you @pavelsavara !

**Pull request merged**